lib/views/login_register/login.dart

```diff
@@ -62,10 +95,11 @@ class LoginScreen extends StatelessWidget {
                             ),
                             child: const TextField(
                               decoration: InputDecoration(
-                                  hintText: 'Email',
-                                  hintStyle: TextStyle(color: Colors.grey),
-                                  border: InputBorder.none,
-                                  labelStyle: TextStyle(color: Colors.black)),
+                                hintText: 'Email',
+                                hintStyle: TextStyle(color: Colors.grey),
+                                border: InputBorder.none,
+                                labelStyle: TextStyle(color: Colors.black),
+                              ),
```

Comment:
(in my opinion, the trailing comma here makes it just slightly easier to follow)